docs/get-started/upgrade-guide.md

```diff
@@ -0,0 +1,20 @@
+# Upgrade Guide
```

Comment:
This isn't final. It's hard for me to get a complete sense of what's changed from `v1/v2` to `v3`. A lot of the materials on the CHANGELOG is documenting changes between `v3` alpha versions which I think shouldn't surface here.